oscar_stripe/facade.py

```diff
@@ -28,17 +31,21 @@ def charge(self,
         description=None,
         metadata=None,
         **kwargs):
+        logger.info("Authorizing payment on order '%s' via stripe" % (order_number))
         try:
             charge_and_capture_together = getattr(settings,
                 "STRIPE_CHARGE_AND_CAPTURE_IN_ONE_STEP", False)
-            return stripe.Charge.create(
-                amount=(total.incl_tax * 100).to_integral_value(),
-                currency=currency,
-                card=card,
-                description=description,
-                metadata=(metadata or {'order_number': order_number}),
-                capture = charge_and_capture_together,
-                **kwargs).id
+            stripe_auth_id = stripe.Charge.create(
+                    amount=(total.incl_tax * 100).to_integral_value(),
+                    currency=currency,
+                    card=card,
+                    description=description,
+                    metadata=(metadata or {'order_number': order_number}),
+                    capture = charge_and_capture_together,
+                    **kwargs
+                ).id
+            logger.info("Authorization of payment on order '%s' via stripe -- SUCCESSFUL" % (order_number))
```

Comment:
Would it be better to just put something like this?

"Authorized payment for order %s via stripe."